plugins/dell-esrt/fu-dell-esrt-plugin.c

```diff
@@ -1,102 +1,27 @@
 /*
  * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
- * Copyright (C) 2017 Dell, Inc.
+ * Copyright (C) 2017 Uefi, Inc.
```

Comment:
still Dell I think :)

Reply:
curious of why this file left alone in this directory? shouldn have migrated into `uefi-esrt`, no?